### DISCUSSION TITLE
Type aliases?

### DISCUSSION THREAD

**Discussion by elrom** [iliekturtles/uom#498] [Q&A]
This example is contrived, but in structs I need to name the actual type and cant rely on type auto deduction. The full type name is of course not very readable (Quanitity<dyn Dimension <... etc), and so I assume there is some clean way to refer to it?
(either the new call is wrong or uom::ratio::percent is the wrong type)

```
use uom::si::{f32::Ratio, ratio::percent};
fn main() {
    let foo: percent = Ratio::new::<percent>(0.0);
}
```

**Comment by chungwong** [comment]
The type for foo is wrong. It is of type `Ratio`.
```rs
let foo: Ratio = Ratio::new::<percent>(0.0);
let foo = Ratio::new::<percent>(0.0);
```
